frontend/catalyst/jax_primitives.py

```diff
@@ -1426,8 +1426,19 @@ def _qfor_lowering(
         loop_operands.append(p)
 
     if apply_reverse_transform:
-        zero = ArithConstantOp(ir.IndexType.get(), 0)
-        one = ArithConstantOp(ir.IndexType.get(), 1)
+        zero_np = np.array(0)
```

Comment:
Is it something that will be reverted in a potential future Jax release (support for InferResultType for Arith)? If yes I would add a comment for an easier revisit. If no, ignore this comment.

Reply:
I don't believe JAX has plans to include the arithmetic dialect in their release. Instead they will continue using StableHLO and only StableHLO.

Reply:
Thanks!